packages/x-date-pickers/src/DesktopTimePicker/tests/DesktopTimePicker.test.tsx

```diff
@@ -80,8 +80,12 @@ describe('<DesktopTimePicker />', () => {
       fireEvent.click(screen.getByRole('option', { name: '09:00 AM' }));
       expect(onChange.callCount).to.equal(1);
       expect(onChange.lastCall.args[0]).toEqualDateTime(new Date(2018, 0, 1, 9, 0));
+      expect(onAccept.callCount).to.equal(0); // onAccept false by default
+      expect(onClose.callCount).to.equal(0); // onCloseOnSelect false by default
```

Comment:
Nitpick: I'm not sure if there is point in duplicating the same comment. 🙈 🤷 
P.S. Adding this suggestion once.
```suggestion
      // closeOnSelect false by default
      expect(onAccept.callCount).to.equal(0);
      expect(onClose.callCount).to.equal(0);
```